### PR TITLE
Update: Replace deprecated web maps with new maps in `Open Map (URL)`

### PR DESCRIPTION
**Pull request by yo1995**
See `common-samples/issues/2832` for details.

This PR replaces the 2 deprecated portal items used in `Open Map (URL)` sample. Other changes

- Removed 1 screenshot
- Cleared redirect field in metadata
- Replaced the thumbnails in assets folder

### CODE REVIEW

File: arcgis-ios-sdk-samples/Maps/Open map (URL)/OpenMapURLViewController.swift [hunk 1]
```diff
@@ -35,12 +35,12 @@ class OpenMapURLViewController: UIViewController {
     }
     
     private let mapModels: [MapAtURL] = [
```
Comment (vquach2404) [inline arcgis-ios-sdk-samples/Maps/Open map (URL)/OpenMapURLViewController.swift:37]: I would consider renaming this to "mapOptions" or something similar. But it's not a big deal; up to you.

**Review by vquach2404 — DISMISSED** [review]
Looks good! Feel free to request others.

**Review by philium — COMMENTED** [review]
The new images appear to be stretched horizontally. Was that intentional? It doesn't look great in my opinion:

image image

**Comment by yo1995** [comment]
Interesting. I didn't noticed that. I took them from the thumbnail images of the portal items:

- https://arcgis.com/home/item.html?id=5be0bc3ee36c4e058f7b3cebc21c74e6
- https://arcgis.com/home/item.html?id=064f2e898b094a17b84e4a4cd5e5f549

Let me recapture these images.

**Comment by yo1995** [comment]
The new thumbnails are taken from the original authoritative sources

- World Terrestrial Ecosystems: https://www.arcgis.com/home/item.html?id=926a206393ec40a590d8caf29ae9a93e
- Recent Hurricanes, Cyclones and Typhoons: https://www.arcgis.com/home/item.html?id=adfe292a67f8471a9d8230ef93294414

**Review by philium — APPROVED** [review]

**Review by vquach2404 — APPROVED** [review]

**Review by saratchandrakarumuri — APPROVED** [review]
looks good

**Pull request merged**